tests/test_modeling_encoder_decoder.py

```diff
@@ -288,38 +287,6 @@ def create_and_check_bert_encoder_decoder_model_labels(
         self.assertEqual(outputs_encoder_decoder[1].shape, (decoder_input_ids.shape + (decoder_config.vocab_size,)))
         self.assertEqual(outputs_encoder_decoder[2].shape, (input_ids.shape + (config.hidden_size,)))
 
-    def create_and_check_bert_encoder_decoder_model_lm_labels(
-        self,
-        config,
-        input_ids,
-        attention_mask,
-        encoder_hidden_states,
-        decoder_config,
-        decoder_input_ids,
-        decoder_attention_mask,
-        lm_labels,
-        **kwargs
-    ):
-        encoder_model = BertModel(config)
-        decoder_model = BertForMaskedLM(decoder_config)
```

Comment:
Can we change to `BertLMHeadModel` in all tests here.

Reply:
Done.
Note that the tests using automodel pick the wrong decoder for now.